tests/app/questionnaire/rules/test_rule_evaluator.py

```diff
@@ -5,7 +5,12 @@
 from freezegun import freeze_time
```

Comment:
Rule evaluator has tests for the different source type, we should do the same for SDS.

Reply:
Added a couple of extra tests, not sure how much extra ill be testing by adding it throughout so just picked a couple if that seems suitable?

Reply:
That is plenty.